app/models/check_list.rb

```diff
@@ -3,6 +3,7 @@ class CheckList < List
   belongs_to :place
   belongs_to :taxon
   belongs_to :source
+  has_one :check_list_place, class_name: "Place", foreign_key: :check_list_id
```

Comment:
This isn't needed since the `List` model has the association, and `CheckList` extends `List`